### PR TITLE
Fix dual-stack metal3-dev-env compatibility

### PR DESCRIPTION
**Pull request by russellb**

### CODE REVIEW

**Comment by metal3ci** [comment]
Build FAILURE, see build http://10.8.144.11:8080/job/dev-tools/1721/

File: 01_install_requirements.sh [hunk 1]
```diff
@@ -13,10 +13,7 @@ if [ -z "${METAL3_DEV_ENV}" ]; then
   sync_repo_and_patch metal3-dev-env https://github.com/metal3-io/metal3-dev-env.git
   METAL3_DEV_ENV="${REPO_PATH}/metal3-dev-env/"
 fi
-
 pushd ${METAL3_DEV_ENV}
-# FIXME: Pin to aafbb89df8df88bc3ecbfe59592149c929432515
-git reset aafbb89df8df88bc3ecbfe59592149c929432515 --hard
```
Comment (stbenjam) [inline 01_install_requirements.sh:19]: I think I'd prefer to keep the pin, but on something newer. Eventually we could automate pushing it forward if OpenShift CI passes.

We need to prove the job's stability and this breakage proves using metal3-dev-env latest is always going to be risky.

Comment (russellb) [reply 01_install_requirements.sh:19]: OK, I'll add this back.

**Comment by metal3ci** [comment]
Build FAILURE, see build http://10.8.144.11:8080/job/dev-tools/1722/

**Comment by russellb** [comment]
> Build FAILURE, see build http://10.8.144.11:8080/job/dev-tools/1722/

doesn't look related, but I'm not positive .. i'm going to check again later to see if CI is passing on any other PRs today

**Comment by russellb** [comment]
Dropped WIP because I *think* this is ready, but CI isn't passing on any PRs right now.  I'd still hold off until a successful deployment is verified in CI or in a dev env.  I'll try again today.

**Comment by stbenjam** [comment]
The OpenShift CI periodic has been passing on and off all weekend, about 1/4 - 1/3 of the time it fails. The reason is worker deployments sporadically fail.

When you look at the worker's console, it failed to DHCP.

Looking at https://github.com/openshift/machine-api-operator/blob/master/pkg/operator/baremetal_pod.go, we start all these containers at once.  I could see there being a race, but I haven't confirmed it by matching up timestamps yet.

**Comment by russellb** [comment]
I got a successful full install on my dev env with these changes.

**Comment by stbenjam** [comment]
Right, the failures are not sporadic. I'd just kick off the Jenkins CI again to confirm this does pass

**Comment by metal3ci** [comment]
Build FAILURE, see build http://10.8.144.11:8080/job/dev-tools/1726/

**Comment by russellb** [comment]
This can just be dropped in favor of #1017

**Pull request closed without merging**